index.html

```diff
@@ -384,9 +384,9 @@ <h2 style="color:white" class="section-heading">Contact Us</h2>
                     <span class="error" id="error_message"></span>
                   </div>
                   <div class="form_group">
-                    <div class="row">
+                    <div class="row" style="margin: 0 auto; width: 656px; text-align: center;" >
```

Comment:
This is totally unnecessary styling. Firstly, always try to keep your markup, your styling, and your styling separate. So, no style tags. Secondly, I don't see how this is helping our case.